ModIds: [14,13,24,25]
JointNames: [joint_platform_neckZ, joint_neckZ_neck, joint_neck_headZ, joint_headZ_head]
MaxAcc: [0.8,0.8,0.8,0.8]

torso_controller:
  type: "JointSplineTrajectoryController"
  joints:
    - joint_platform_neckZ
    - joint_neckZ_neck
    - joint_neck_headZ
    - joint_headZ_head
  gains:
    joint_platform_neckZ: {p: 1200.0, i: 120, d: 12.0}
    joint_neckZ_neck: {p: 1200.0, i: 120, d: 12.0}
    joint_neck_headZ: {p: 1000.0, i: 100, d: 10.0}
    joint_headZ_head: {p: 1000.0, i: 100, d: 10.0}
  joint_trajectory_action_node:
    joints:
      - joint_platform_neckZ
      - joint_neckZ_neck
      - joint_neck_headZ
      - joint_headZ_head
    constraints:
      goal_time: 0.6
      joint_platform_neckZ:
        goal: 0.02
      joint_neckZ_neck:
        goal: 0.02
      joint_neck_headZ:
        goal: 0.02
      joint_headZ_head:
        goal: 0.02
